src/control/server/ioserver/exec.go

```diff
@@ -50,8 +51,9 @@ type (
 
 	// Runner starts and manages an instance of a DAOS I/O Server
 	Runner struct {
-		Config *Config
-		log    logging.Logger
+		Config  *Config
+		log     logging.Logger
+		started uint32
```

Comment:
I guess this is a uint32 specifically to take advantage of the atomic library. It was counterintuitive at first - since it is used as a bool. The goal is to avoid using mutexes, I'm guessing?

Reply:
yes for the simple use case I think a mutex is unnecessary